.github/workflows/gradle.yml

```diff
@@ -8,10 +8,10 @@ jobs:
 
     steps:
     - uses: actions/checkout@v2
-    - name: Set up JDK 8
+    - name: Set up JDK 11
       uses: actions/setup-java@v1
       with:
-        java-version: 8
+        java-version: 11
```

Comment:
can i get a rationale behind using java 11 and not 8?

Reply:
ask @notOM3GA (who suggested it)
i didnt know what to use, assumed they knew better than me, and changed it
i know that you are smart, and i have no justification, so should i change it back, or what version do you suggest

Reply:
> java 8 is deprecated and is end of life. java 11 provides many qol features/changes and a significant performance boost. everything will remain backwards compatible.

Ah, OK, so here is the justification (didnt see this)

Reply:
ok

Reply:
I'd stay on java 8, as most people who would use this mod would most likely use java 8

Reply:
Java has great backwards compatibility. It will still work on Java 8.